### ISSUE TITLE
Project imported with QtCreator has missing src files

### ISSUE THREAD

**Issue by hamoid** [openframeworks/openFrameworks#4987]
```
git clone https://github.com/npisanti/ofxPDSP.git
git clone https://github.com/danomatika/ofxMidi.git
git clone https://github.com/npisanti/ofxSIMDFloats.git
```

I import ```ofxPDSP/example-midi_polysynth``` into QtCreator and click run, but it fails to compile. [See error screenshot here](https://github.com/npisanti/ofxPDSP/issues/2#issuecomment-195808952).

If I type ```make Debug``` on the command line, the program compiles successfully.

I'm on 64bit ArchLinux, OF 0.9.3 from GitHub, QtCreator 3.6.0.

**Update:** by looking at the .qbs project file, I see two files from src/ are missing:

        files: [
            'src/main.cpp',
            'src/ofApp.cpp',
            'src/ofApp.h',
        ]

The two missing files are voice.cpp and voice.h.

**Comment by hamoid** [comment]
Looking at the import wizard, it looks to me like it always includes the same 3 files found in src/.

At the bottom of this document https://doc.qt.io/qtcreator/creator-project-wizards-json.html it mentions a scanner, which builds a list of files found in a folder, and I don't see it used in the wizard (maybe I missed it). It looks like the solution to this issue :)

**Comment by arturoc** [comment]
yes the project importer is kind of limited at the moment, it only adds the standard files. i didn't see any option for reading the contents of the folder when i created the wizard. i might have missed it or it might be something they've added afterwards, in any case it would be a great addition to have. if you want to take a look? otherwise i'll try to implement it when i have a moment

**Comment by hamoid** [comment]
I spent a long time trying to figure it out, but the documentation is very limited. In 

    /usr/share/qtcreator/templates/wizards/projects/vcs/*/wizard.json

they use the Scanner generator, but I have no idea where is the list of files accessible. I thought of using JavaScript to get the list of files placing that into "options", but then I also could not find any documentation about the syntax for those JS calls. For instance I see Util.filename, Util.isDirectory, Vcs.isConfigured but I don't know what that Util or Vcs refers to. Knowing that I could do something like Util.listFiles(...) and include that list inside the generated qbs file.
